html_css/intermediate_css/enhancing-design-with-css.md

```diff
@@ -53,4 +53,4 @@ This section contains questions for you to check your understanding of this less
 * Which property would you use to add scrollbars to an element?
 * Which property would you use to add a shadow behind an element?
 * Which property would you use to create rounded corners on an element?
-* How would you use border-radius to make a circular element?
\ No newline at end of file
+* How would you use border-radius to make a circular element?
```

Comment:
Should be the last thing. I believe we need a new line at the end of the lesson files.